### PR TITLE
Hide step actions for response if in studio

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -** 
Added a has_real_user prop based on whether the runtime is bound to a real user.

This is now linked such that nothing below the response shows up if it is true.

JIRA: [AU-58](https://openedx.atlassian.net/browse/AU-58)

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

View an ORA in studio and verify the Save, file upload, and actions (submit button) don't show up
Verify that they do for the LMS view.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/xblock/openassessmentblock.py [hunk 1]
```diff
@@ -746,10 +746,9 @@ def in_studio_preview(self):
             bool
 
         """
-        # When we're running in Studio Preview mode, the XBlock won't provide us with a user ID.
-        # (Note that `self.xmodule_runtime` will still provide an anonymous
-        # student ID, so we can't rely on that)
-        return self.scope_ids.user_id is None
+        # When we're running in Studio Preview mode, the XBlock won't
+        # be tied to a real user
+        return self.xmodule_runtime.get_real_user is None
```
Comment (nsprenkle) [inline openassessment/xblock/openassessmentblock.py:751]: Follow the pattern from other instances of calling the `xmodule_runtime`
```suggestion
        if hasattr(self, 'xmodule_runtime'):
            return self.xmodule_runtime.get_real_user is None  # pylint: disable=no-member
        return False
```

**Review by nsprenkle — COMMENTED** [review]
Looks good except for failing quality test. Have added suggested fix following patterns throughout the repo. Will test locally.

**Review by nsprenkle — APPROVED** [review]
I've never been so excited to _not_ see something on the page.

File: openassessment/xblock/openassessmentblock.py [hunk 2]
```diff
@@ -751,6 +751,18 @@ def in_studio_preview(self):
         # student ID, so we can't rely on that)
         return self.scope_ids.user_id is None
 
+    @property
+    def in_studio(self):
```
Comment (jansenk) [inline openassessment/xblock/openassessmentblock.py:755]: I feel uneasy about this for some reason, and I think we should instead be explicit and just call this "has_no_real_user" or something like that. It's a blurry, kinda backwards way of checking if we're in studio. It works, but I think we'd be safer just calling this what it is, and plus, it makes sense down the line too. Why do we care about not submitting in studio? Because there's no real user. Cut out the middle step and hide the button if there's no user. Does that make sense? I'm kinda rambling.

**Comment by jansenk** [comment]
Does this need an associated template change? Was that already done?

**Review by nsprenkle — DISMISSED** [review]
RE @jansenk, looks like template updates got lost in your recent changes. Need to fix that.

**Review by jansenk — APPROVED** [review]
LGTM

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍

**Review by mattcarter — APPROVED** [review]
LGTM 👍
